homeassistant/components/waze_travel_time/__init__.py

```diff
@@ -147,38 +153,90 @@ async def async_get_travel_times(
             real_time=realtime,
             alternatives=3,
         )
+        _LOGGER.debug("Got routes: %s", routes)
+
+        incl_routes: list[CalcRoutesResponse] = []
+
+        def should_include_route(route: CalcRoutesResponse) -> bool:
+            if len(incl_filters) < 1:
+                return True
+            for street_name in route.street_names:
```

Comment:
Maybe a bit more convoluted.
I wonder if the following isn't preferred in terms of performance.

```python
    result = any(street_name in incl_filters or "" in incl_filters for street_name in route.street_names)
    if not result:
         _LOGGER.debug(...)
```

Reply:
Looks good